src/reputation/serializers/bounty_serializer.py

```diff
@@ -75,6 +88,58 @@ def get_hubs(self, bounty):
 
         return []
 
+    def get_category(self, bounty):
+        """Return category hub if it exists"""
+        if bounty.unified_document:
+            category = bounty.unified_document.hubs.filter(
+                namespace=Hub.Namespace.CATEGORY
+            ).first()
+            if category:
+                return SimpleHubSerializer(category).data
+        return None
+
+    def get_subcategory(self, bounty):
+        """Return subcategory hub if it exists"""
+        if bounty.unified_document:
+            subcategory = bounty.unified_document.hubs.filter(
+                namespace=Hub.Namespace.SUBCATEGORY
+            ).first()
+            if subcategory:
+                return SimpleHubSerializer(subcategory).data
+        return None
+
+    def get_journal(self, bounty):
+        """Return journal hub if it exists"""
+        if not bounty.unified_document:
+            return None
+
+        journal_hubs = [
+            hub
+            for hub in bounty.unified_document.hubs.all()
+            if hub.namespace == Hub.Namespace.JOURNAL
+        ]
+
+        if not journal_hubs:
+            return None
+
+        researchhub_journal = None
+        for hub in journal_hubs:
+            if int(hub.id) == int(settings.RESEARCHHUB_JOURNAL_ID):
+                researchhub_journal = hub
+                break
+
+        # Use ResearchHub Journal if found, otherwise use the first journal
+        journal_hub = researchhub_journal or journal_hubs[0]
+
+        if journal_hub:
+            return {
+                "id": journal_hub.id,
+                "name": journal_hub.name,
+                "slug": journal_hub.slug,
+                "image": journal_hub.hub_image.url if journal_hub.hub_image else None,
+            }
+        return None
```

Comment:
This looks more than just "display logic". And since the source is also modified in https://github.com/ResearchHub/researchhub-backend/pull/2904, I wonder if this should be part of the unified document model so the logic determining the actual journal hub can be reused - with the actual display logic remaining here. @koutst